src/db_impl.cc

```diff
@@ -603,18 +631,23 @@ void TitanDBImpl::OnCompactionCompleted(
       file->FileStateTransit(BlobFileMeta::FileEvent::kCompactionCompleted);
     }
 
+    uint64_t delta = 0;
     for (const auto& bfs : blob_files_size) {
+      delta += bfs.second;
       // blob file size < 0 means discardable size > 0
       if (bfs.second >= 0) {
         continue;
       }
+      delta += -bfs.second;
```

Comment:
The logic around `delta` doesn't look right to me. Line 636 should be remove I think.

Reply:
(how careless am i)